From 1e938d6ce4fd7f411d347e76fe086665c6e0caf4 Mon Sep 17 00:00:00 2001
From: ryanking13 <def6488@gmail.com>
Date: Thu, 7 Dec 2023 00:31:22 +0900
Subject: [PATCH 1/1] Remove fasteners

---
 pyproject.toml | 1 -
 zarr/sync.py   | 2 --
 2 files changed, 3 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 292bfddd..f0d8b78c 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -14,7 +14,6 @@ requires-python = ">=3.8"
 dependencies = [
     'asciitree',
     'numpy>=1.20,!=1.21.0',
-    'fasteners',
     'numcodecs>=0.10.0',
 ]
 dynamic = [
diff --git a/zarr/sync.py b/zarr/sync.py
index 49684a51..da0b86ec 100644
--- a/zarr/sync.py
+++ b/zarr/sync.py
@@ -2,8 +2,6 @@ import os
 from collections import defaultdict
 from threading import Lock
 
-import fasteners
-
 
 class ThreadSynchronizer:
     """Provides synchronization using thread locks."""
-- 
2.29.2.windows.2

